File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //declare height variable
    int height ;
    do
    {
      height = get_int("Height: ");
    } while (height < 1 || height > 8 );

    //prints out (height) rows
    for (int row = 0; row < height; row++)
    {
        //prints out __ spaces
        //row 0, 7 spaces
        //row 1, 6 spaces
        //... row 7, 0 spaces
        for (int space = (height - row - 1); space > 0; space--)
        {
           printf(" ");
        }

        //prints out bricks
        //row 0, 1 brick
        //row 1, 2 bricks
        //... row 7, 8 bricks
        for (int brick = 0; brick < row + 1; brick++)
        {
           printf("#");
        }

        //prints out gap for middle
        printf("  ");

        for (int brick = 0; brick < row + 1; brick++)
        {
           printf("#");
        }
       printf("\n");

    }

}

File: sentimental-mario-more/mario.py

from cs50 import get_int


def main():
    height = get_height()
    for row in range(height):  # Row starts at 0 [0, 1, 2, 3], if height is 4
        for space in range(height - row - 1, 0, -1):  # Space stops before 0
            print(" ", end="")
        for brick in range(row + 1):
            print("#", end="")
        print(" " * 2, end="")
        for brick in range(row + 1):
            print("#", end="")
        print()


def get_height():
    while True:
        h = get_int("height: ")
        if h > 0 and h < 9:
            break
    return h


main()
